sound/mpd/Makefile

```diff
@@ -101,104 +98,90 @@ define Package/mpd-avahi-service/conffiles
 /etc/avahi/services/mpd.service
 endef
 
-TARGET_CFLAGS += -ggdb3
-TARGET_LDFLAGS += -Wl,-rpath-link=$(STAGING_DIR)/usr/lib $(if $(ICONV_FULL),-liconv)
-EXTRA_CXXFLAGS += $(if $(CONFIG_GCC_VERSION_4_8),-std=gnu++11,-std=gnu++14)
+EXTRA_CXXFLAGS += -std=gnu++14
 
 CONFIGURE_ARGS += \
 	$(call autoconf_bool,CONFIG_IPV6,ipv6) \
 	--disable-debug \
 	--disable-documentation \
 	--disable-test \
-	--disable-werror \
-	\
+	--disable-aac \
+	--disable-adplugin \
 	--disable-ao \
+	--disable-audiofile \
 	--disable-bzip2 \
+	--disable-cdio-paranoia \
 	--disable-fluidsynth \
 	--disable-wildmidi \
 	--disable-gme \
 	--enable-inotify \
 	--disable-icu \
-	--disable-eventfd \
 	--disable-iso9660 \
 	--disable-jack \
 	--disable-roar \
 	--disable-libwrap \
 	--disable-lsr \
+	--disable-mad \
 	--disable-mikmod \
 	--disable-modplug \
 	--disable-mpc \
-	--disable-mpg123 \
+	--disable-nfs \
 	--disable-openal \
 	--disable-opus \
 	--disable-pulse \
 	--disable-sidplay \
+	--disable-smbclient \
+	--disable-sndfile \
 	--disable-solaris-output \
 	--disable-sqlite \
+	--disable-systemd-daemon \
 	--disable-lame-encoder \
 	--disable-twolame-encoder \
 	--disable-shine-encoder \
+	--disable-vorbis-encoder \
 	--enable-wave-encoder \
 	--disable-wavpack \
+	--disable-webdav \
 	--disable-wildmidi \
 	--disable-zzip \
 	--with-zeroconf=no \
 	--disable-soxr \
-	\
 	--enable-curl \
-	--enable-flac \
 	--enable-httpd-output \
-	$(call autoconf_bool,CONFIG_BUILD_PATENTED,mad) \
 	$(call autoconf_bool,CONFIG_AUDIO_SUPPORT,alsa) \
 	--enable-tcp \
-	--enable-un \
-
-CONFIGURE_VARS += \
-	FLAC_CFLAGS="$(TARGET_CFLAGS) -I$(STAGING_DIR)/usr/include/FLAC" \
-	FLAC_LIBS="$(TARGET_LDFLAGS) -lFLAC" \
-	$(if $(CONFIG_BUILD_PATENTED),MAD_CFLAGS="$(TARGET_CFLAGS)") \
-	$(if $(CONFIG_BUILD_PATENTED),MAD_LIBS="$(TARGET_LDFLAGS) -lmad") \
+	--disable-sndio \
+	--disable-haiku \
+	--disable-iconv \
 
 ifeq ($(BUILD_VARIANT),full)
 
   CONFIGURE_ARGS += \
 	--enable-upnp \
-	$(call autoconf_bool,CONFIG_BUILD_PATENTED,aac) \
-	--enable-audiofile \
-	--enable-fifo \
 	--enable-ffmpeg \
+	--disable-flac \
 	--enable-id3 \
 	--enable-mms \
-	--enable-flac \
+	--disable-mpg123 \
 	--enable-pipe-output \
 	--enable-recorder-output \
-	--disable-shout \
-	--enable-sndfile \
-	--enable-vorbis \
-	--disable-vorbis-encoder \
-	--with-tremor=yes \
-
+	--enable-shout \
+	--disable-vorbis
 endif
 
 ifeq ($(BUILD_VARIANT),mini)
 
   # oggflac is not compatible with tremor
   CONFIGURE_ARGS += \
 	--disable-upnp \
-	--disable-aac \
-	--disable-audiofile \
 	--disable-fifo \
 	--disable-ffmpeg \
 	--disable-id3 \
 	--disable-mms \
-	--disable-pipe-output \
```

Comment:
Why are these 2 lines going away?  If they're changing, shouldn't they change to an explicit --enable-xyzzy?